internal/infrastructure/kubernetes/proxy_deployment.go

```diff
@@ -140,6 +140,22 @@ func expectedProxyContainers(infra *ir.Infra) ([]corev1.Container, error) {
 		},
 	}
 
+	for _, listener := range infra.Proxy.Listeners {
```

Comment:
dont you need to delete the port config on L130-140 ?

Reply:
sure